datamade/just-spaces
# Add CRUD views for CensusAreas

## Overview

Connects #177. This PR creates Create, List, Edit, and Deactivate views for CensusAreas. For now, the Create and Edit views only display the `name` field, meaning new CensusAreas cannot actually be created. This PR also adds an `is_active` attribute to CensusAreas in order to use them with our deactivate method.

## Testing Instructions

 * Run migrations
 * Click on "Admin" in the sidebar and select "Census Areas"
 * Click around to each view and make sure they're all visible and connected in intuitive ways consistent with the rest of the site
* Navigate to a submitted survey detail page, add a new ACS-compatible chart, and check that a link to the CensusArea list view is displayed in the help text under that field
